plugins/source/aws/resources/services/cloudwatchlogs/data_protection_policies.go

```diff
@@ -17,7 +17,7 @@ func dataProtectionPolicy() *schema.Table {
 		Description: `https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_GetDataProtectionPolicy.html`,
 		Resolver:    fetchDataProtectionPolicy,
 		Multiplex:   client.ServiceAccountRegionMultiplexer(tableName, "logs"),
-		Transform:   transformers.TransformWithStruct(&cloudwatchlogs.GetDataProtectionPolicyOutput{}, transformers.WithPrimaryKeys("LogGroupIdentifier"), transformers.WithSkipFields("ResultMetadata")),
+		Transform:   transformers.TransformWithStruct(&cloudwatchlogs.GetDataProtectionPolicyOutput{}, transformers.WithSkipFields("ResultMetadata")),
```

Comment:
No need for double PKs, and also `LogGroupIdentifier` has a tendency to come up empty, it could also come empty for deleted/archived data protection entries, we simply don't know.